.changeset/wet-cycles-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/jest': major
+---
+
+`@emotion/jest/serializer`'s main purpose is compatibility with Jest's `snapshotSerializers` option, so it no longer has a default export. You can import `createSerializer` from the root entry (`@emotion/jest`) and create your own serializer if needed.
```

Comment:
I assume `import * as serializer from '@emotion/jest/serializer'` would work?

Reply:
And if so, in this changeset and the one for enzyme stuff, explain that they're individual exports now.

Reply:
> I assume import * as serializer from '@emotion/jest/serializer' would work?

Yes, it is used like that somewhere in our tests.

> And if so, in this changeset and the one for enzyme stuff, explain that they're individual exports now.

Done - gonna go ahead and merge this PR, but if you have any other comments then please leave them here and I will make sure to address them.